src/camera/CircularCamera.ts

```diff
@@ -148,20 +231,23 @@ class CircularCamera extends Camera {
       }, []);
 
       flicking.renderer.movePanelElementsToStart(passedPanels);
-
       this._circularOffset += this._calcPanelAreaSum(passedPanels);
     }
 
+    flicking.renderer.render();
+
     return super.lookAt(pos);
   }
 
   protected _applyTransform(): void {
     const el = this._el;
     const flicking = getFlickingAttached(this._flicking, "Camera");
 
+    const actualPosition = this._position - this._alignPos - this._offset + this._circularOffset;
+
     el.style[this._transform] = flicking.horizontal
-      ? `translate(${-(this._position - this._alignPos + this._circularOffset)}px)`
-      : `translate(0, ${-(this._position - this._alignPos + this._circularOffset)}px)`;
+      ? `translate(${-actualPosition}px)`
+      : `translate(0, ${-actualPosition}px)`;
   }
```

Comment:
It repeats with the code [here](https://github.com/naver/egjs-flicking/pull/420/files#diff-1b8ee851a75d0e755767cd49b1e3330cc8066793bf7660a7d6faa532b6541b48R302-R304). How about make a method?

Reply:
Okay, I'll refactor that.